jewelbots_nRF51822_board_v100/cores/JWB_nRF51822/Timer.h

```diff
@@ -12,7 +12,7 @@ extern "C"{
     Timer();
     ~Timer();
     void pause(uint32_t milliseconds);
-
+    uint32_t millis(void);
```

Comment:
I love a more descriptive name. The reason I chose for `millis` is that the Arduino library provides a [`millis`](https://www.arduino.cc/en/Reference/Millis) function. I am not sure what the stance of the Jewelbots libary is regarding, but I could imagine that they would want to stay as close to arduino standard languages as possible.

Having said that, in that case it would be better tot move it off the `Timer` struct and to a proper function.